lib/onelogin/ruby-saml/saml_message.rb

```diff
@@ -88,11 +88,11 @@ def valid_saml?(document, soft = true)
       # @param saml [String] The deflated and encoded SAML Message
       # @return [String] The plain SAML Message
       #
-      def decode_raw_saml(saml)
+      def decode_raw_saml(saml, settings)
```

Comment:
instead the whole setting object, can we just add message_max_bytesize param by default MAX_BYTE_SIZE?

Reply:
@pitbulk I'm pretty sure we could, but this follows the [existing pattern used for `encode_raw_saml`](https://github.com/onelogin/ruby-saml/blob/master/lib/onelogin/ruby-saml/saml_message.rb#L111) (the inverse method).  Do you want me to change this method even though it would then differ from `encode_raw_saml`?

Reply:
ohh you are right, let's then keep it as it is